pkgs/development/tools/doctl/default.nix

```diff
@@ -2,15 +2,29 @@
 
 buildGoPackage rec {
   name = "doctl-${version}";
-  version = "1.3.1";
-  rev = "a57555c195d06bc7aa5037af77fde0665ad1231f";
+  version = "${major}.${minor}.${patch}";
+  major = "1";
+  minor = "5";
+  patch = "0";
+  rev = "66f10023539b5e0e27485a34048366ca1133872f";
   goPackagePath = "github.com/digitalocean/doctl";
 
+  excludedPackages = "\\("
+    + stdenv.lib.concatStringsSep "\\|" ([ "doctl-gen-doc" "install-doctl" "release-doctl" ])
+    + "\\)";
```

Comment:
It should :joy: I'm learning the nix expression language lol, I like to complicate things :joy: